docusaurus/docs/guides/kubernetes/workload-tunneling/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Connecting Kubernetes Workloads with OpenZiti
+---
+
+This guide will help you discover the best strategy and tool for connecting a Kubernetes workload with OpenZiti. This overview page links to separate child pages for each strategy and tool.
+
+```mermaid
+graph TD; 
+  A{Workload connects to an </br>OpenZiti Service?};
+  A -- Yes --> B{Pod sidecar or </br>node daemonset?}; 
+  A -- No --> K{OpenZiti Router </br>is installed?};
+  K -- Yes --> J(OpenZiti Router provides </br>cluster ingress for </br>OpenZiti Services);
+  K -- No --> C{Nginx </br>is </br>installed?};
+  C -- Yes --> D(Nginx container with </br>OpenZiti proxy module </br>provides cluster ingress for </br>OpenZiti Services);
+  C -- No --> E(ziti-host </br>reverse proxy pod</br>provides cluster ingress </br>for OpenZiti Services); 
+  B -- NODE --> F(tun proxy daemonset </br>provides the same DNS and</br>access to all pods on selected nodes);
+  B -- POD --> G{Workload uses DNS or </br>static IP to connect to </br>the OpenZiti Service?};
+  G -- YES --> H(transparent sidecar </br>&#40iptables tproxy&#41</br> provides DNS and </br>IP interception to the pod);
+  G -- NO --> I(loopback sidecar </br>&#40tcp proxy&#41</br>binds a local port to </br>each OpenZiti Service);
+  click D "../../securing-apis/aks-api-with-nginx-ziti-module" "OpenZiti Nginx Module";
+  click E "./kubernetes-host/" "ziti-host";
+  click F "./kubernetes-daemonset/" "node proxy";
+  click H "./kubernetes-sidecar/" "transparent sidecar";
+  click I "#loopback-proxy-sidecar" "loopback sidecar";
+```
+
+## Strategies and Solutions
+
+As you can see in the chart above, there are several strategies and solutions for connecting a Kubernetes workload with OpenZiti. The following sections describe each strategy and solution in more detail. These are organized into two main categories: cluster ingress to workloads with OpenZiti, and intercepting workload egress with OpenZiti.
```

Comment:
> with OpenZiti. The following sections describe each strategy and solution in more detail. These are organized into two main categories: cluster ingress to workloads with OpenZiti, and intercepting workload egress with OpenZiti

There are three "with OpenZiti" in these 20-30 words :) prolly can remove them all?

Reply:
In fact, reread this whole doc finding specifically for " with OpenZiti" and consider removing all of them? :)